src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, serif;
+  font-weight: 400;
+
+  transition: all linear $main-during;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__image {
+    display: flex;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    height: 134px;
+    margin-inline: 3px;
+    margin-bottom: 40px;
+    padding: 0;
+    background-origin: border-box;
+  }
+
+  &__text {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+    color: $main_accent-color;
+
+    transition: all linear $main-during;
+  }
+
+  &__text--product-code {
+    color: $secondary-color;
+    margin-bottom: 16px;
+  }
+
+  &__text--reviews {
+    color: $main_accent-color;
+    align-self: center;
+  }
+
+  &__row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__row--feedback {
+    margin-bottom: 24px;
+  }
+
+  &__row--price {
+    margin-bottom: 16px;
+  }
+
+  &__text--price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__text--costs {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main_accent-color;
+  }
+
+  &__link {
+    background-color: $blue_accent-color;
+    border: 1px solid $blue_accent-color;
+
+    height: 40px;
+    box-sizing: border-box;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    border-radius: 5px;
+
+    color: white;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+
+    transition: all linear $main-during;
+    &:hover {
+      background-color: #fff;
+      color: $blue_accent-color;
+    }
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
```

Comment:
The selector '&:hover &__title' might not work as intended. Ensure that the nested hover effect is correctly targeting the intended elements. Consider using '&:hover .card__title' if the title is a direct child of the card.